server/admin.api.js

```diff
@@ -100,6 +100,27 @@ router.get('/admin/finduncompressed', mw.adminOnly, (req, res)=>{
 		});
 });
 
+/* Cleans `<script` and `</script>` from the "text" field of a brew */
+router.put('/admin/clean/script/:id', asyncHandler(HomebrewAPI.getBrew('admin', false)), async (req, res)=>{
+	console.log(`[ADMIN] Cleaning script tags from ShareID ${req.params.id}`);
+
+	function cleanText(text){return text.replaceAll(/(<\/?s)cript/gi, '');};
+
+	const brew = req.brew;
+
+	const properties = ['text', 'description', 'title'];
+	properties.forEach((property)=>{
+		brew[property] = cleanText(brew[property]);
+	});
+	// Tag cleaning is commented out as it is impossible to enter a script tag in tags
+	// brew.tags = cleanText(brew.tags.join('\n')).split('\n');
```

Comment:
Do you expect this commented section to ever be needed in the future or should it just be removed?

Reply:
Comment removed in latest commit.